packages/big-design/src/components/List/GroupSeparator/ListGroupSeparator.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { memo } from 'react';
+
+import { HR } from '../../Typography';
+
+export const ListGroupSeparator: React.FC<React.LiHTMLAttributes<HTMLLIElement>> = memo(() => (
+  <li aria-hidden={true}>
+    <HR />
+  </li>
+));
```

Comment:
We need to add some of the similar features `ListGroupHeader` has for a11y.

```suggestion
export const ListGroupSeparator: React.FC<React.LiHTMLAttributes<HTMLLIElement>> = memo(() => (
  <li tabIndex={-1} onMouseDown={preventFocus} aria-hidden={true}>
    <HR />
  </li>
));

function preventFocus(event: React.MouseEvent<HTMLLIElement, MouseEvent>) {
  event.preventDefault();
}
```